dashboard/src/modules/components/TableComponent/common-components.jsx

```diff
@@ -34,7 +36,7 @@ export const LoginHint = (props) => {
           variant="link"
           key="login-hint-button"
           className="alertHelpText"
-          onClick={() => navigate(`/${redirect}`)}
+          onClick={() => navigate(`${redirect}`)}
```

Comment:
This `navigate()` is from `useNavigate()`, not from the `Outlet` context...that's not a problem?  (And, the relative path target is OK?)

Reply:
Updated